BUILD.bazel

```diff
@@ -303,6 +303,23 @@ cc_binary(
     ],
 )
 
+cc_binary(
+    name = "spirv-dump",
```

Comment:
We might want a better name. This name make is seem like the spirv will be dumped. I'll try to think of one.

Reply:
`spirv-src` -> bad for the directory structure (`tools/src`)
`spirv-source` -> ok-ish, but too specific to extract source.
Seemed like I was doing some kind of objdump but for spirv, so spirv-dump. But agree.
`spirv-lift` -> conceptually what we do, but bad as we don't lift, just read debug infos.
`spirv-extract` -> too close in the mind to extraction tools
`spirv-peek` -> like dotPeek? but not well known.
'spirv-rev' -> maybe? but too generic. (reverse?)

Reply:
How about a longer name: `spirv-source-dump` or `spirv-dump-source`?

Reply:
renamed to `spirv-objdump`, let me know if you have another idea 😊